src/components/Filter/Filter.tsx

```diff
@@ -3,20 +3,22 @@ import { useTranslation } from 'react-i18next';
 
 import Dropdown from '../Dropdown/Dropdown';
 import Button from '../Button/Button';
-import useBreakpoint, { Breakpoint } from '../../hooks/useBreakpoint';
 
 import styles from './Filter.module.scss';
 
+import useBreakpoint, { Breakpoint } from '#src/hooks/useBreakpoint';
+
 type Props = {
   name: string;
   value: string;
   valuePrefix?: string;
   defaultLabel: string;
   options: string[];
+  dropDown?: boolean;
```

Comment:
Maybe it's better to call this forceDropdown or something like that, because the filter can sometimes be a dropdown still even when it's false.